### ISSUE TITLE
Fix logging across modules

### ISSUE THREAD

**Issue by kmcquade** [salesforce/policy_sentry#268]
Right now, we use click_log to set the log formatter. However, I noticed that the logger only shows up when it is in the command file - like `policy_sentry.command.write_policy`, but not `policy_sentry.writing.sid_group`

For example, running write policy with the minimize command like this:

```bash
policy_sentry write-policy --input-file examples/yml/crud-with-wildcard.yml --minimize 0 -v debug
```

In the command above, **there are no debug messages that show up** but a lot of them **should** show up. 

For example, the write-policy command leverages the `policy_sentry.write_policy.sid_group.SidGroup` class here, but if you look at the actual `sid_group.py` file, you will see that there are a lot of other lines with `logger.debug()`. But these logging messages do not show up. 

For example:

https://github.com/salesforce/policy_sentry/blob/0166584e93882f9d256247306847a3a9fd4f94af/policy_sentry/writing/sid_group.py#L178-L182

If you have any experience with setting the Python log formatter across multiple modules, I would really appreciate help on this one.

**Comment by reetasingh** [comment]
@kmcquade I added a PR for cloudspanning https://github.com/salesforce/cloudsplaining/pull/135
if that is merged I can take this up. please assign it to me

**Comment by kmcquade** [comment]
@reetasingh thanks for addressing this! The one in cloudsplaining looks good; I merged it.

**Comment by kmcquade** [comment]
This was fixed by #269. Closing
